taxonomies/categories.txt

```diff
@@ -42769,12 +45566,15 @@ fr:Sirops d'érable foncés, Sirops d'érable foncé, Sirop d'érable D
 nl:Donkere esdoornstropen
 
 <en:Simple syrups
-en:Molasses
-es:Melazas de caña, Melazas, Melaza, Miel de caña, Melaza de caña
+en:Molasses, Cane molasses
+es:Melazas de caña, Melazas, Melaza, Miel de caña, Melaza de caña, Mélasse de canne
```

Comment:
"Mélasse de canne" should be on the fr: line instead of es:

Reply:
Sloppy copy / paste, I guess. Thanks!